internal/executor/taskfilter/taskfilter.go

```diff
@@ -63,8 +63,10 @@ func extractLabels(s string) (result []string) {
 	return
 }
 
-func matchLabels(soughtLabels, actualLabels []string) (numMatchedLabels int) {
-	for _, soughtLabel := range soughtLabels {
+func containsAll(desiredLabels, actualLabels []string) bool {
+	var numMatchedLabels int
+
+	for _, soughtLabel := range desiredLabels {
```

Comment:
Maybe rename to `desiredLabel` for consistency?

Reply:
See 68552bc.